README.md

```diff
@@ -1,318 +1,194 @@
 
-## Introduction
-BuckleScript is a JavaScript backend for [the OCaml compiler](https://ocaml.org/).
-
-You can try
-[BuckleScript in the browser](http://bloomberg.github.io/bucklescript/js-demo/),
-edit the code on the left panel, and see the generated JS on the right
-panel instantly.
-
-Users of BuckleScript can write type-safe, high performance OCaml code,
-and deploy the generated JavaScript in any platform with a JavaScript
-execution engine.
-
-Each OCaml module is mapped to a corresponding JavaScript module, and names are preserved so that:
-
-1. The stack trace is preserved, so the generated code is debuggable with or without a source map.
-2. Modules generated by BuckleScript can be used directly from other JavaScript code. For example, you can call the `List.length` function from the OCaml standard library `List` module from JavaScript.
-
-### A simple example
-
-```ocaml
-let port = 3000
-let hostname = "127.0.0.1"
-let create_server http  = 
-  let server = http##createServer begin fun [@bs] req  resp  -> 
-      resp##statusCode #= 200;
-      resp##setHeader "Content-Type" "text/plain";
-      resp##_end "Hello world\n"
-    end
-  in
-  server##listen port hostname  begin fun [@bs] () -> 
-    Js.log ("Server running at http://"^ hostname ^ ":" ^ Pervasives.string_of_int port ^ "/")
-  end 
-
-
-let () = 
-  create_server Http_types.http
-```
-Generated by BuckleScript
-```js
-'use strict';
-var Pervasives = require("bs-platform/lib/js/pervasives");
-var Http       = require("http");
-
-var hostname = "127.0.0.1";
-
-function create_server(http) {
-  var server = http.createServer(function (_, resp) {
-        resp.statusCode = 200;
-        resp.setHeader("Content-Type", "text/plain");
-        return resp.end("Hello world\n");
-      });
-  return server.listen(3000, hostname, function () {
-              console.log("Server running at http://" + (hostname + (":" + (Pervasives.string_of_int(3000) + "/"))));
-              return /* () */0;
-            });
-}
-
-create_server(Http);
-```
-
-You can download the example [here](https://github.com/bloomberg/bucklescript-addons/tree/master/examples/node-http-server):
-
-
-As you can see, there is no name mangling in the generated code, the function `string_of_int` is from a normal OCaml module `Pervasives`, `Pervasives.string_of_int` is directly callable from other JavaScript code.
+## Try BuckleScript
 
-The attribute `[@bs]` allows BuckleScript to support JS's uncurried calling convention in addition to OCaml's curried calling convention.
+You can try BuckleScript directly [in your browser](http://bloomberg.github.io/bucklescript/js-demo/). Write OCaml in the left panel and
+watch as it instantly compiles to JavaScript shown in the right panel.
 
+## Installing BuckleScript
 
+#### Pre-Requisites
 
-## Disclaimer
+* Standard C/C++ toolchain
+* Node 4 or newer
```

Comment:
node is not a dependency, if people build from source, and we don't depend on c++ toolchain